src/app/code/community/Divante/VueStorefrontIndexer/Model/Elasticsearch/Client/Builder.php

```diff
@@ -52,7 +51,7 @@ private function getHost(array $options)
     {
         $scheme = 'http';
 
-        if (isset($options['enable_https_mode'])) {
+        if (isset($options['enable_https_mode']) && $options['enable_https_mode'] !== false) {
             $scheme = 'https';
         } elseif (isset($options['schema'])) {
             $scheme = $options['schema'];
```

Comment:
I've updated the typo but I left the `if (isset($options['enable_https_mode']) && $options['enable_https_mode'] !== false)` for also checking the value of `enable_https_mode` not just its existens in case it is `false` – do you agree with that?

Reply:
OK